### ISSUE TITLE
bug in ha-selector-number: slider can not handle min: 0

### ISSUE THREAD

**Issue by philippwaller** [home-assistant/frontend#13867]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

the component "ha-selector-number" cannot select the minimum value in slider mode if it is `min: 0`.

![number selector bug](https://user-images.githubusercontent.com/1090452/192143541-950ef08e-c605-4b51-b6ff-b4055686a8b4.gif)

this is the selctor configuration, I am using:

```
blueprint:
  name: Close shutter at sundown
  description: Close shutter at sundown
  domain: automation
  input:
    cover_target_position:
      name: Cover Target Position
      description: This is the position the cover will be set to when the automation is triggered.
      selector:
        number:
          min: 0
          max: 100
          unit_of_measurement: "%"
          mode: slider
...
```

### Describe the behavior you expected

The min value should be selectable. ~~In "box" mode, the component works as expected.~~

### Steps to reproduce the issue

1. create a blueprint
2. create a selector with the following configuration:
```
number:
  min: 0
  max: 100
  unit_of_measurement: "%"
  mode: slider
```

### What version of Home Assistant Core has the issue?

2022.9.6 - Frontend 20220907.2

### In which browser are you experiencing the issue with?

Chrome Version 105.0.5195.125

### Which operating system are you using to run this browser?

macOS Monterey 12.4

**Comment by GrumpyMeow** [comment]
I don't have much experience with Home Assistant development, but i've tried to reproduce your issue.
Unfortunately i'm unable to reproduce the issue. Are you sure that the issue is not specifically related with your blueprint?
Nothing in the browser console-logs and/or Core-logs?
![image](https://user-images.githubusercontent.com/12073499/192384604-00c15a43-cb1e-41d7-8b78-ae7787202eeb.png)
![image](https://user-images.githubusercontent.com/12073499/192384699-45baeb94-daf8-440a-b351-8d447fb6f0e5.png)

**Comment by philippwaller** [comment]
@GrumpyMeow: Thanks for your feedback. I can reliably reproduce the error in Chrome and Safari. I have updated my example with a complete blueprint. 

Unfortunately, there is no console message related to this issue.

**Comment by silamon** [comment]
The issue cannot be reproduced in the gallery, but if you're testing with the given example in a demo instance, you can track it down to the line below: 
https://github.com/home-assistant/frontend/blob/aa673774a8c13d159ee8a082e7aff51fa6cc196b/src/panels/config/automation/blueprint-automation-editor.ts#L187

**Comment by philippwaller** [comment]
Bug was solved with #14053. Thanks a lot!
